### PR TITLE
Changes status code for cstor pool recommendation

### PR DESCRIPTION
**Pull request by amitbhatt818**
- Changed status code for GET and POST request

Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

### CODE REVIEW

File: litmus/director/tcid-ppcp01-create-cstor-pool/test.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@
             force_basic_auth: yes
             body_format: json
             body: '{"clusterId":"{{ cluster_id }}", "raidGroupConfig":{"groupDeviceCount":1, "raidType":"stripe"}}'
-            status_code: 201
+            status_code: 200
```
Comment (shovanmaity) [inline litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: can we put a range of valid status code?

Comment (amitbhatt818) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: We can put multiple status codes . But I am not sure about the range of status codes @shovanmaity

Comment (amitbhatt818) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: Should I add 201 also in the status code @shovanmaity

Comment (shovanmaity) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: 200 – 299

Comment (amitbhatt818) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: @shovanmaity I think we cannot add range of status code .But we can add two or more status code ex: `status_code: 200, 201, 203`

Comment (shovanmaity) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml:69]: Ok add 200, 201 and 202

Comment (amitbhatt818) [reply litmus/director/tcid-ppcp01-create-cstor-pool/test.yml]: @shovanmaity Added 201 and 202 as status code PTAL

**Pull request merged**